usaspending_api/download/helpers/elasticsearch_download_functions.py

```diff
@@ -86,11 +88,39 @@ def _populate_download_lookups(cls, filters: dict, download_job: DownloadJob, si
         )
 
         created_obj_list = DownloadJobLookup.objects.bulk_create(download_lookup_obj_list, batch_size=10000)
```

Comment:
Realizing this when I was chatting with you earlier, but how does this code (which is running on the BD instance) have a connection to the the master PG db? Does it?
I guess, is it's connection _always_ to the master DB... and it's only the `psql` command that gets handed the read-only download replica connection?

Just asking: does this bulk create insert into master? or try to insert into the read-only replica?